### ISSUE TITLE
Acquisition and production of Webvision 1.0 training set and test set

### ISSUE THREAD

**Issue by lijichang** [shengliu66/ELR#13]
Your work is very inspiring.

I tried to run Webvision Dataset on your code but failed as the accuracy result is only 50% at top-1. I suspect I have a problem with the acquisition and production of the data set.
I wonder if you can provide details about how to get this data and how to get ImagenetVal and how to make them. For example, how can I get imagenet_val.txt?

Thanks a lot.
